core/src/test/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/AttributeComponentSpec.scala

```diff
@@ -693,6 +693,8 @@ class AttributeComponentSpec extends TestDriverComponentWithFlatSpecAndMatchers
           s"a new attribute when $description1 changes to $description2" in {
           attributeTestFunction.run(attribute1, attribute2)
         }
+      case x =>
+         throw new Exception("this is unexpected")
```

Comment:
lol. Is this because it's pattern matching on a list? Maybe change 
```scala
attributeTestData.combinations(2).flatMap(x => List(x, x.reverse)).foreach {
```
to 
```scala
attributeTestData.combinations(2).flatMap(x => (x, x.reverse)).foreach {
```
so you can pattern match on the pair exhaustively?

Reply:
`attributeTestData.combinations(2).flatMap(x => (x, x.reverse))` doesn't compile

Reply:
Ah sorry, I misunderstood what this code did and that it was a `List` because of `combinations(2)`. Would be amazing if scala got dependenct types.